src/port.js

```diff
@@ -232,6 +247,18 @@ export default class Port {
 				.resolve(handlerResult)
 				.then((val) => {
 					me.sendMessage(`res.${requestType}`, { id: w.id, val: val });
+				})
+				.catch((e) => {
```

Comment:
Given the stack is useless on client-side, maybe we should debug log the real error here?